quora-service/src/main/java/com/upgrad/quora/service/entity/UserAuthTokenEntity.java

```diff
@@ -0,0 +1,170 @@
+package com.upgrad.quora.service.entity;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+
+@Entity
+@Table(name = "user_auth_tokens", schema = "qoura")
```

Comment:
Schema name should be "quora". Typo error
Table name should be "user_auth". Please check

Reply:
Issue fixed in the latest commit. Good observation, thanks!

Reply:
:)